docker-cadg-apache/README.md

```diff
@@ -0,0 +1,19 @@
+# CADG Apache Docker Image
+
+This utilizes version 2.4 of the Apache httpd server. The configuration file is extremely basic at present and will need to be built. As such, this has been tagged as prototype for the time being.
+
+To build locally
+
+docker build -t image-name:tag .
```

Comment:
Wrap in triple backticks.
Change to "docker build -t cadg-apache:tag ."

Reply:
:+1: